mapstory/apps/organizations/forms.py

```diff
@@ -0,0 +1,40 @@
+from django import forms
+
+
+class BasicInformation(forms.Form):
+    """
+    Collects basic information for an Organization
+        - Org Name
+        - Org Slogan
+        - City
+        - Country
+        - Display Photo
+        - About
+    """
+    name = forms.CharField(required=True)
+    slogan = forms.CharField(required=True)
+    city = forms.CharField(required=False)
+    country = forms.CharField(required=False)
+    image = forms.ImageField(required=False)
+    about = forms.CharField(widget=forms.Textarea, required=True)
+
+
+class LinksAndSocialMedia(forms.Form):
+    """
+    LINKS & SOCIAL MEDIA
+    --------------------
+        - URLs
+        - Facebook
+        - Twitter
+        - LinkedIn
+        - Github
+        - Instragram
+    """
+    url0 = forms.URLField(required=False)
```

Comment:
Do we only want three URLs as maximum for orgs to list?

Reply:
Created issue #1090 for this